llvm/lib/Target/AMDGPU/SIInsertWaitcnts.cpp

```diff
@@ -1692,22 +1662,19 @@ bool SIInsertWaitcnts::generateWaitcntInstBefore(MachineInstr &MI,
       // load). We also need to check WAW dependency with saved PC.
       Wait = AMDGPU::Waitcnt();
 
-      int CallAddrOpIdx =
-          AMDGPU::getNamedOperandIdx(MI.getOpcode(), AMDGPU::OpName::src0);
-
-      if (MI.getOperand(CallAddrOpIdx).isReg()) {
+      const auto *CallAddrOp = TII->getNamedOperand(MI, AMDGPU::OpName::src0);
```

Comment:
Could make this a ref?
```suggestion
      const auto &CallAddrOp = *TII->getNamedOperand(MI, AMDGPU::OpName::src0);
```

Reply:
Done